src/server/services/appwrite.ts

```diff
@@ -15,80 +15,78 @@ export function getUniqueID() {
   return ID.unique();
 }
 
-export function getOAuthProvider(provider: keyof typeof oAuthProviders) {
-  return oAuthProviders[provider];
-}
+export class AppWrite {
+  clientInstance;
```

Comment:
is there any risk
https://appwrite.io/docs/tutorials/nextjs-ssr-auth/step-3

![image](https://github.com/user-attachments/assets/486b3618-6f09-4827-81e9-265225d02262)

Reply:
will check this